qanta/wikipedia/cached_wikipedia.py

```diff
@@ -183,8 +206,13 @@ def __getitem__(self, key: str):
             if raw:
                 if len(raw) > 1:
                     log.info("%i pages for %s" % (len(raw), key))
+
+                wikidata = self.get_wikidata(unformatted_key)
```

Comment:
Could you attach some sample code showing using the object as well as sample output of the stored file?